manifests/etcdquorumguard/deployment.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: etcd-quorum-guard
+  namespace: kube-system
+spec:
+  replicas: 3
```

Comment:
I think it should be possible to teach mco to scale up / decide the replica count based on number of master node.

Reply:
Agreed beyond 4.1; for 4.1, it has been decided to only support 3 masters.